### ISSUE TITLE
Cloudflare redirect list

### ISSUE THREAD

**Issue by lvets** [cloudflare/terraform-provider-cloudflare#1729]
### Current Terraform and Cloudflare provider version

```
❯ terragrunt -v
terragrunt version v0.38.0

❯ terraform -v
Terraform v1.1.9
on darwin_amd64

Your version of Terraform is out of date! The latest version
is 1.2.3. You can update by downloading from https://www.terraform.io/downloads.html

❯
```

### Description

Is it possible to add redirect lists? These can be found under Account -> Configurations -> Lists -> Create new list -> Content type: Redirect.

### Use cases

Manage redirect lists in Terraform :)

### Potential Terraform configuration

```hcl
...
```

### References

I think the upstream Go library supports this? See https://github.com/cloudflare/cloudflare-go/blob/master/list.go.

**Comment by Cyb3r-Jak3** [comment]
@jacobbednarz I believe this has been implemented here: https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/resource_cloudflare_list.go

**Comment by jacobbednarz** [comment]
closed by #1700

**Comment by lvets** [comment]
Forgot to close bug. This is working, thanks!
